rules/collections/annotations/java.yaml

```diff
@@ -34,3 +34,9 @@ collections:
     patterns:
       - "GET|POST"
     tags:
+
+  - id: Collections.Annotation.JAX-RS
+    name: JAX-RS Annotation
+    patterns:
+      - "(?i)(javax.ws.rs).(GET|PUT|POST|DELETE|HEAD|OPTIONS).*"
```

Comment:
please check if we match method fullnames for collection points. AFAIK its just the annotation name.

Reply:
I will check this.

Reply:
@HiteshMah-Jan we are only checking the 'name' and not the method fullname. You are right. We are using `cpg.annotation.name` to figure out matches.

Reply:
This rule would not work then. Can I just change it to `".*(GET|PUT|POST|DELETE|HEAD|OPTIONS).*"`, this way it'll match case-sensitive names for these ? @HiteshMah-Jan